diagnostic.md

```diff
@@ -32,23 +36,29 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+GETTER:
+$('#header').val()
+$('#header').text()
+SETTER:
+$('#header').val('NEW PREZ')
+$('#header').text('NEW PREZ')
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is what the browser creates using our code and the browser's fillers (what it's programmed to automatically fill in).
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.